chart/kubeapps/templates/tls-secrets.yaml

```diff
@@ -14,19 +14,4 @@ data:
 ---
 {{- end }}
 {{- end }}
-{{- if and .Values.ingress.tls (not .Values.ingress.certManager) }}
-{{- $ca := genCA "kubeapps-ca" 365 }}
-{{- $cert := genSignedCert .Values.ingress.hostname nil (list .Values.ingress.hostname) 365 $ca }}
-apiVersion: v1
-kind: Secret
-metadata:
-  name: {{ printf "%s-tls" .Values.ingress.hostname }}
-  namespace: {{ .Release.Namespace }}
-  labels: {{ include "kubeapps.labels" . | nindent 4 }}
-type: kubernetes.io/tls
-data:
-  tls.crt: {{ $cert.Cert | b64enc | quote }}
-  tls.key: {{ $cert.Key | b64enc | quote }}
-  ca.crt: {{ $ca.Cert | b64enc | quote }}
-{{- end }}
```

Comment:
I do find this useful for development purposes.
Instead, I'd recommend to add a `ingress.selfSigned` parameter to explicitly indicate when the self-signed cert should be created.

Reply:
> I do find this useful for development purposes.

You mean the generation right? In what situation when doing development do you find it useful? That is, if I am doing development and using TLS, I'd prefer to work with a cert that is trusted by my computer, which is exactly what I was doing here before this was added. But yep, perhaps others prefer differently, but I'm not keen on development options like this being in the chart by default if you switch TLS on (and used outside dev envs).

> Instead, I'd recommend to add a `ingress.selfSigned` parameter to explicitly indicate when the self-signed cert should be created.

OK, see what you think, I've updated to that.

Reply:
ping @juan131 ?

Reply:
Just a side note for you all to know: from now on, the changes performed in this chart -won't - be automatically pushed to the bitnami/chart repository (as required as part of https://github.com/kubeapps/kubeapps/issues/2704)

Reply:
ping @juan131 Is there any reason you see for this not to land? It's now got exactly the same condition on the tls secret generation as you've landed in https://github.com/bitnami/charts/pull/6235/files

> Just a side note for you all to know: from now on, the changes performed in this chart -won't - be automatically pushed to the bitnami/chart repository (as required as part of #2704)

Yep, I've removed the chart bump from this PR because of that, but made me realise, we probably want a `devel` version in our local `Chart.yaml` otherwise it could get confusing when people  use our devel chart and it says it's the actual published version. I'll do that separately ad we can discuss there.

Reply:
I'm okay with the current approach! Thanks @absoludity